drivers/hmis/app/models/hmis/group_viewable_entity.rb

```diff
@@ -10,5 +10,27 @@ class GroupViewableEntity < GrdaWarehouse::GroupViewableEntity
 
     belongs_to :access_group, class_name: '::Hmis::AccessGroup'
     belongs_to :entity, polymorphic: true
+
+    scope :includes_project, ->(project) do
+      pags = project.project_groups
+      where(entity_type: project.class.name, entity_id: project.id).
+        or(includes_data_source(project.data_source)).
+        or(includes_organization(project.organization)).
+        or(includes_data_source(project.organization.data_source)).
```

Comment:
Line 19 can be removed - the project data source and organization data source are the same